### ISSUE TITLE
v1.76 Alpha 1 no console option in group

### ISSUE THREAD

**Issue by cjszoo** [mRemoteNG/mRemoteNG#845]
There are console session options on connections, but if you inherit all, there isn't an option to set from the group to inherit. It looks like allot of options are missing that inherit, so i may be looking in wrong place.
thanks,

**Comment by sparerd** [comment]
Can you explain a bit more what you mean? When you set a connection to inherit a setting, it takes the setting value from its direct parent (whatever folder it is in).

**Comment by cjszoo** [comment]
Correct,

The console setting isn’t available in the parent.

I went to the alpha for the RDC 2.7 import, which works great. 

I’ve tried creating a new folder and with protocol on RDP most rdp options are not present.

OFC, won’t be the first time it’s my fault, can you reproduce this behavior?

 

Here are the only settings in the parent folder, newly created and not a subfolder.

 



 

Here is a connection under this folder, set to inherit all. I have that option as well as many others that aren’t available in the parent.

I cannot inherit that and get console option.

**Comment by sparerd** [comment]
I think the pics you provided were removed when the email was converted to a post on Github. If the parent folder isn't a subfolder (it's parent is the Connections node), then all the properties should indeed be available. If they aren't it's likely that the folder has inheritance turned on and is trying to inherit from the Connections root node (which is definitely a bug). Take a look at #807 if that seems to be what is going on.

**Comment by cjszoo** [comment]
It looks like the same thing.

On the root (connections) I have inherit set off. Then on the next subfolder, I have no control over inheritance, its greyed out. No matter what I set the root (connections) to, there is no change to my subfolder or RDP connections below that.

To clarify…. I am running the zip version. I imported a RDC 2.7 file and moved those under this subfolder under the root (connections). I have created a new subfolder under connections and created an RDP connection in that to see if it had anything to do with the RDC import.

Thanks again,
